sos/report/__init__.py

```diff
@@ -445,7 +445,8 @@ def _get_hardware_devices(self):
                 'fibre': self._get_fibre_devs()
             },
             'network': self._get_network_devs(),
-            'namespaced_network': self._get_network_namespace_devices()
+            'namespaced_network': self._get_network_namespace_devices(),
+            'fstype': self._get_devices_by_fstype()
```

Comment:
Nit: I think naming this `filesystem(s)` is a bit more in line with naming for things plugin authors might access.

Reply:
While I have no issue giving it name filesystems, it might be even more misleading than fstypes. The blkid (libblkid) provides also other types such as lvm2, crypto, many RAIDs, swap, drdb, bcache ...  full list available via blkid -k.

Reply:
That's a fair point. `fstypes` is fine by me then, as I mentioned just a nitpick anyways.